extensions/EXT/EGL_EXT_query_reset_notification_strategy.txt

```diff
@@ -0,0 +1,80 @@
+Name
+
+	EXT_query_reset_notification_strategy
+
+Name Strings
+
+	EGL_EXT_query_reset_notification_strategy
+
+Contributors
+
+	Simon Zeni <simon.zeni@collabora.com>
+
+Contact
+
+	Simon Zeni <simon.zeni@collabora.com>
+
+Status
+
+	Draft
+
+Version
+
+	Version 0.1, 2023-11-01
+
+Number
+
+	EGL Extension #152
+
+Extension Type
+
+	Client
+
+Dependencies
+
+	Requires EGL 1.4 and EXT_create_context_robustness.
+
+	The extension is written against the wording of the EGL 1.5
+	specification, but may be implemented against 1.4.
+
+Overview
+
+	This extension allows the user to query the reset notification strategy
+	of a given context using the function eglQueryContext. This extension
+	complements the EXT_create_context_robustness one in order to be able
```

Comment:
```suggestion
	complements EXT_create_context_robustness in order to be able
```

This is still a bit obtuse, since this extension has nothing to do with creating contexts.

Maybe something like: 

This extension complements EXT_create_context_robustness and enables an application or framework to determine an existing context's reset notification strategy in order to enable it to create a compatible shared context.

Reply:
I reworded it based on your feedback, thanks